java/client/src/main/java/glide/api/RedisClient.java

```diff
@@ -78,4 +87,118 @@ public CompletableFuture<Object> customCommand(String[] args) {
                 Command.builder().requestType(Command.RequestType.CUSTOM_COMMAND).arguments(args).build();
         return commandManager.submitNewCommand(command, BaseClient::handleObjectResponse);
     }
+
+    /**
+     * Ping the Redis server.
+     *
+     * @see <a href="https://redis.io/commands/ping/">redis.io</a> for details.
+     * @returns the String "PONG"
```

Comment:
Would it be preferable to specify that the method does not return a string but instead a completable future that resolves to a string? Same question for the other javadocs in this file. Also, note that javadocs use @return instead of @returns
```suggestion
     * @return a completable future that returns the String "PONG"
```

Reply:
```suggestion
     * @return A completable future that returns server response
```
Server implementation maybe changed